botorch/models/approximate_gp.py

```diff
@@ -386,6 +370,33 @@ def __init__(
 
         self.to(train_X)
 
+    def init_inducing_points(self, X, num_inducing) -> Tensor:
+        with torch.no_grad():
+            inducing_points = _select_inducing_points(X, self.model.covar_module, num_inducing,
+                                                      self._input_batch_shape)
+            self.model.variational_strategy.base_variational_strategy.inducing_points.copy_(
+                inducing_points.clone()
+            )
+        return inducing_points
+
+
+def _select_inducing_points(X, covar_module, num_inducing, input_batch_shape) -> Tensor:
+    train_train_kernel = covar_module(X)
+
+    if train_train_kernel.ndimension() > 2:
+        # TODO this assumes X is the same along the batch dimensions
+        train_train_kernel = train_train_kernel.evaluate_kernel()[0]
+
+    inducing_points = _pivoted_cholesky_init(
+        X, train_train_kernel, max_length=num_inducing
+    )
+
+    # previous batch handling didn't work in the multitask setting
```

Comment:
Thanks for the fix!
```suggestion
    # make sure batch handling works in the multitask setting
```

Reply:
I guess I'm suggesting to indicate what we're doing here, not stating what didn't work previously :)

Reply:
sure thing :)